modules/cms/controllers/ThemeOptions.php

```diff
@@ -43,6 +46,7 @@ public function __construct()
         parent::__construct();
 
         $this->pageTitle = 'cms::lang.theme.settings_menu';
+        $this->registerLocalization($this->getDirName());
```

Comment:
Having the registration of theme language files here means that these translations would only be available specifically on this controller for specifically the currently active theme. @bennothommo what do you think about making them more widely available in the backend in general? Is there any benefit to that?

I would probably say that they definitely shouldn't be made available in the frontend given that might encourage developers to use this as a feature to localize themes rather than support with RainLab.Translate, which would be a worse UX overall for the end users; however I'm not 100% sure if locking them just to this particular context is the right option or not. 

@mjauvin @bennothommo any thoughts?

Reply:
I see what you mean @LukeTowers but I personally think this is perfect as is since the intended use is to allow theme options to be localizable WITHOUT requiring an extra plugin to do this (self contained theme).

What other controller do you have in mind that could use this?

Reply:
@LukeTowers I think the only other area where there would be any need for localisation from the a theme would be the Syntax Parser fields which get rendered into a form in the Static Pages plugin - not sure if that's already being localised in some way already?

Otherwise, the Theme Options controller would be the only place where localisation would be needed.

I agree with you that the chief method of translation on the theme content itself should be done through the Translate plugin.

Reply:
That's a good point re: the syntax parser fields. @GinoPane can you take a stab at that?

Reply:
So what did you guys decide? I'm not sure I know anything about syntax parser fields. Is it another controller?

Reply:
@GinoPane it's in the RainLab.Pages plugin, but what should work the best is to just do a simple App::isRunningInBackend() around the language registration code for all present themes in the CMS ServiceProvider.

Reply:
Hm, I didn't plan to change any of the plugins. This is purely for vanilla system backend.
Do you suggest to surround this line with the `isRunningInBackend` condition?

![image](https://user-images.githubusercontent.com/3897579/98208961-38c14600-1f4f-11eb-9a59-dd947fce8a79.png)

Reply:
Yes, but also move it to the CMS ServiceProvider rather than the theme options controller

Reply:
Ok, moved it